api/turing/config/example.yaml

```diff
@@ -14,6 +14,8 @@ AuthConfig:
 # Batch ensembler job configurations
 BatchEnsemblingConfig:
   Enabled: true
+  # Note that {{.Project}} and {{.Job}} is required.
```

Comment:
Can we add more details here on what templating vars are supported exactly (is it only these 2?) and what they will be replaced with? Like `{{.Project}}` and `{{.Job}}` for project name and job name, respectively.

Reply:
Yep ok